ci/jenkins/jobs/projects.yaml

```diff
@@ -584,7 +584,20 @@
           test_name: eks-conformance-net-policy
           node: antrea-cloud
           description: 'This is EKS conformance and network policy test for antrea.'
-          parameters: []
+          parameters:
+          - string:
+              default: https://github.com/vmware-tanzu/antrea/
+              description: The repository to checkout Antrea for this test.
+              name: ANTREA_REPO
+              trim: 'true'
+          - string:
+              default: master
+              description: The branch or SHA commit ID to checkout and build Antrea for this test.
+              name: ANTREA_VERSION
```

Comment:
maybe `ANTREA_VERSION` is a bit confusing. What about `ANTREA_GIT_REVISION`?

Reply:
Done